src/RamCondition.h

```diff
@@ -454,15 +454,15 @@ class RamEmptinessCheck : public RamCondition {
  */
 inline std::vector<std::unique_ptr<RamCondition>> toConjunctionList(const RamCondition* condition) {
     std::vector<std::unique_ptr<RamCondition>> list;
-    std::stack<const RamCondition*> stack;
+    std::queue<const RamCondition*> queue;
```

Comment:
Could you rename `queue` to be more informative?

`toCheck` or `conditionsToCheck`, perhaps?

`list` could be `conjunctionList`